cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -415,6 +430,117 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
             return withVoid(serviceOperations.reconcile(namespace, kafkaCluster.getHeadlessServiceName(), kafkaHeadlessService));
         }
 
+        Future<ReconciliationState> kafkaReplicaServices() {
+            int replicas = kafkaCluster.getReplicas();
+            List<Future> serviceFutures = new ArrayList<>(replicas);
+
+            for (int i = 0; i < replicas; i++) {
+                serviceFutures.add(serviceOperations.reconcile(namespace, KafkaCluster.externalServiceName(name, i), kafkaCluster.generateExternalService(i)));
+            }
+
+            return withVoid(CompositeFuture.join(serviceFutures));
+        }
+
+        Future<ReconciliationState> kafkaBootstrapRoute() {
+            Future future = Future.succeededFuture();
```

Comment:
Remove this and make the final `return` be `return withVoid(Future.succeededFutre())`.